src/styles.scss

```diff
@@ -257,9 +257,29 @@
   }
 
   .ClaimableBalances {
-    .Table {
-      --table-min-width: 500px;
-      width: auto;
```

Comment:
I think we should keep this (with updated value to accommodate for the extra column). Do you think it looks better full width?

Reply:
I did change the style.scss to make it full width. I think they both look good :) I will change set it back to its original value with an updated value to accommodate the extra column, as per your suggestion.